doc/BACKUP.md

```diff
@@ -0,0 +1,413 @@
+in-channel activity*.
+This is in contrast to onchain payments, where you have to tell everyone
+about each and every payment and have it recorded on the blockchain,
+leading to scaling problems (you have to push data at everyone) and
+privacy problems (everyone knows every payment you were ever involved in).
+
+Unfortunately, this removes a property that onchain users are so used
+to, they react in surprise when learning about this removal.
+Your onchain activity is recorded in all archival fullnodes, so if you
+forget all your onchain activity because your storage got fried, you
+just go redownload the activity from the nearest archival fullnode.
+
+But in Lightning, since *you* are the only one storing all your
+financial information, you ***cannot*** recover this financial
+information anywhere else.
+
+This means that on Lightning, you **have to** responsibly back up your
+financial information yourself, using various processes and automation.
+
+The discussion below assumes that you know where you put your
+`$LIGHTNINGDIR`, and you know the directory structure within.
+By default your `$LIGHTNINGDIR` will be in `~/.lightning/${COIN}`.
+For example, if you are running `--mainnet`, it will be
+`~/.lightning/bitcoin`.
+
+`hsm_secret`
+------------
+
+You need a copy of the `hsm_secret` file regardless of whatever backup
+strategy you use.
+This is just 32 bytes, and you can do something like the below and
+write the hexadecimal digits a few times on a piece of paper:
+
+    cd $LIGHTNINGDIR
+    xxd hsm_secret
+
+You can re-enter the hexdump into a text file later and use `xxd` to
+convert it back to a binary `hsm_secret`:
+
+    cat > hsm_secret_hex.txt <<HEX
+    00: 30cc f221 94e1 7f01 cd54 d68c a1ba f124
+    10: e1f3 1d45 d904 823c 77b7 1e18 fd93 1676
+    HEX
+    xxd -r hsm_secret_hex.txt > hsm_secret
+    chmod 0400 hsm_secret
+
+Notice that you need to ensure that the `hsm_secret` is only readable by
+the user, and is not writable, as otherwise `lightningd` will refuse to
+start.
+Hence the `chmod 0400 hsm_secret` command.
+
+Alternately, if you are deploying a new node that has no funds and
+channels yet, you can generate BIP39 words using any process, and
+create the `hsm_secret` using the `hsmtool generatehsm` command.
+If you did `make install` then `hsmtool` is installed as
+`lightning-hsmtool`, else you can find it in the `tools/` directory
+of the build directory.
+
+    lightning-hsmtool generatehsm hsm_secret
+
+Then enter the BIP39 words, plus an optional passphrase.
+
+You can regenerate the same `hsm_secret` file using the same BIP39
+words, which again, you can back up on paper.
+
+Recovery of the `hsm_secret` is sufficient to recover any onchain
+funds.
+Recovery of the `hsm_secret` is necessary, but insufficient, to recover
+any in-channel funds.
+To recover in-channel funds, you need to use one or more of the other
+backup strategies below.
+
+Database File Backups
+---------------------
```

Comment:
IMHO the methods should be sorted in descending order of desirability, putting this type first gives absolutely the wrong impression.